### ISSUE TITLE
Saved files not loading

### ISSUE THREAD

**Issue by STyx2909** [triplea-game/triplea#8850]
Application error is logged as: Window manager warning: Invalid WM_TRANSIENT_FOR window 0x0 specified for 0x26000de

### How can the problem be recreated?
Most or many of saved and autosaved games cannot be reopened. Some are. It happened with tutorial too.

### Do you have any ideas for an expected fix?
From a search for a similar error issue, this could be a VRAM problem. 8-/

```
(...)
I've gotten a little further, seems to be some kind of Gnome Shell issue. I am getting this error the second the Burp UI goes black:

**gnome-shell[1401]: Window manager warning: Invalid WM_TRANSIENT_FOR window 0x0 specified for XXX**

peter | Last updated: Apr 28, 2020 04:48PM UTC

Solved. It wasn't a burp issue at all. **Had to increase video memory on the VMWare host.** Sorry for the run around, and thanks for all the help!
(...)
```

### Attach a Save Game
[20200214 test.zip](https://github.com/triplea-game/triplea/files/5977724/20200214.test.zip)
Today's tests.

Game Version: 2.5.22294

### Is there anything else we should know?
Sure Ubuntu 20.04 up to date. Java (JRE) 11.0.10, Old Asus K61ic with 4Gb RAM and Nvidia GT220M with 1Gb VRAM
and "canberra-gtk-module" which is a 32bit library is not installed on my system.

**Comment by beelee1** [comment]
@STyx2909 right on so you got it fixed ? If so you can close this issue.

**Comment by STyx2909** [comment]
**I did not fix it.**
Ran a test with 1.9.0 which is the latest official Ubuntu supported version.

Same problem appears. But we have more details. So, I took a quick look to check repeating patterns (with 3 attempts to read files) and extracted this:
```
15:09:06 java: 	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:90)
15:09:06 java: 	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
15:09:06 java: 	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:109)
15:09:06 java: 	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:113)
15:09:06 java: 	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:124)
15:09:06 java: 	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:203)
15:09:06 java: 	at games.strategy.engine.framework.GameRunner$1.dispatchEvent(GameRunner.java:330)
15:09:06 java: 	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:742)
15:09:06 java: 	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:85)
15:09:06 java: 	at java.base/java.security.AccessController.doPrivileged(Native Method)
15:09:06 java: 	at java.desktop/java.awt.EventQueue$5.run(EventQueue.java:743)
15:09:06 java: 	at java.desktop/java.awt.EventQueue$5.run(EventQueue.java:745)
15:09:06 java: 	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:95)
15:09:06 java: 	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:85)
15:09:06 java: 	at java.base/java.security.AccessController.doPrivileged(Native Method)
15:09:06 java: 	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:715)
15:09:06 java: 	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:721)
15:09:06 java: 	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:772)
15:09:06 java: 	at java.desktop/java.awt.Component.dispatchEvent(Component.java:4843)
15:09:06 java: 	at java.desktop/java.awt.Window.dispatchEventImpl(Window.java:2772)
15:09:06 java: 	at java.desktop/java.awt.Container.dispatchEventImpl(Container.java:2307)
15:09:06 java: 	at java.desktop/java.awt.LightweightDispatcher.dispatchEvent(Container.java:4488)
15:09:06 java: 	at java.desktop/java.awt.LightweightDispatcher.processMouseEvent(Container.java:4547)
15:09:06 java: 	at java.desktop/java.awt.LightweightDispatcher.retargetMouseEvent(Container.java:4918)
15:09:06 java: 	at java.desktop/java.awt.Component.dispatchEvent(Component.java:4843)
15:09:06 java: 	at java.desktop/java.awt.Container.dispatchEventImpl(Container.java:2321)
15:09:06 java: 	at java.desktop/java.awt.Component.dispatchEventImpl(Component.java:5011)
15:09:06 java: 	at java.desktop/java.awt.Container.processEvent(Container.java:2263)
15:09:06 java: 	at java.desktop/java.awt.Component.processEvent(Component.java:6400)
15:09:06 java: 	at java.desktop/javax.swing.JComponent.processMouseEvent(JComponent.java:3342)
15:09:06 java: 	at java.desktop/java.awt.Component.processMouseEvent(Component.java:6635)
15:09:06 java: 	at java.desktop/java.awt.AWTEventMulticaster.mouseReleased(AWTEventMulticaster.java:297)
15:09:06 java: 	at org.pushingpixels.substance.internal.utils.RolloverButtonListener.mouseReleased(RolloverButtonListener.java:124)
15:09:06 java: 	at java.desktop/javax.swing.plaf.basic.BasicButtonListener.mouseReleased(BasicButtonListener.java:279)
15:09:06 java: 	at java.desktop/javax.swing.DefaultButtonModel.setPressed(DefaultButtonModel.java:262)
15:09:06 java: 	at java.desktop/javax.swing.DefaultButtonModel.fireActionPerformed(DefaultButtonModel.java:405)
15:09:06 java: 	at java.desktop/javax.swing.AbstractButton$Handler.actionPerformed(AbstractButton.java:2308)
15:09:06 java: 	at java.desktop/javax.swing.AbstractButton.fireActionPerformed(AbstractButton.java:1967)
15:09:06 java: 	at games.strategy.engine.framework.startup.ui.GameSelectorPanel.lambda$setupListeners$1(GameSelectorPanel.java:228)
15:09:06 java: 	at games.strategy.engine.framework.startup.ui.GameSelectorPanel.selectGameFile(GameSelectorPanel.java:373)
15:09:06 java: 	at games.strategy.engine.framework.startup.mc.GameSelectorModel.load(GameSelectorModel.java:97)
15:09:06 java: 	at games.strategy.engine.framework.GameDataManager.loadGame(GameDataManager.java:68)
15:09:06 java: 	at games.strategy.engine.framework.GameDataManager.loadGame(GameDataManager.java:87)
15:09:06 java: 	at games.strategy.engine.framework.GameDataManager.loadGameInSerializationFormat(GameDataManager.java:153)
15:09:06 java: 	at java.base/java.io.ObjectInputStream.readObject(ObjectInputStream.java:451)
15:09:06 java: 	at java.base/java.io.ObjectInputStream.readObject(ObjectInputStream.java:493)
15:09:06 java: 	at java.base/java.io.ObjectInputStream.readObject0(ObjectInputStream.java:1679)
15:09:06 java: 	at java.base/java.io.ObjectInputStream.readOrdinaryObject(ObjectInputStream.java:2196)
15:09:06 java: 	at java.base/java.io.ObjectInputStream.readSerialData(ObjectInputStream.java:2325)
15:09:06 java: 	at java.base/java.io.ObjectStreamClass.invokeReadObject(ObjectStreamClass.java:1175)
15:09:06 java: 	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
15:09:06 java: 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
15:09:06 java: 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
15:09:06 java: 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
15:09:06 java: 	at games.strategy.engine.data.GameData.readObject(GameData.java:117)
15:09:06 java: 	at java.base/java.io.ObjectInputStream.defaultReadObject(ObjectInputStream.java:629)
15:09:06 java: 	at java.base/java.io.ObjectInputStream.defaultReadFields(ObjectInputStream.java:2464)
15:09:06 java: 	at java.base/java.io.ObjectInputStream.readObject0(ObjectInputStream.java:1679)
15:09:06 java: 	at java.base/java.io.ObjectInputStream.readOrdinaryObject(ObjectInputStream.java:2196)
15:09:06 java: 	at java.base/java.io.ObjectInputStream.readSerialData(ObjectInputStream.java:2358)
15:09:06 java: 	at java.base/java.io.ObjectInputStream.defaultReadFields(ObjectInputStream.java:2464)
15:09:06 java: 	at java.base/java.io.ObjectInputStream.readObject0(ObjectInputStream.java:1679)
15:09:06 java: 	at java.base/java.io.ObjectInputStream.readOrdinaryObject(ObjectInputStream.java:2196)
15:09:06 java: 	at java.base/java.io.ObjectInputStream.readSerialData(ObjectInputStream.java:2325)
15:09:06 java: 	at java.base/java.io.ObjectStreamClass.invokeReadObject(ObjectStreamClass.java:1175)
15:09:06 java: 	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
15:09:06 java: 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
15:09:06 java: 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
15:09:06 java: 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
15:09:06 java: 	at com.google.common.collect.ImmutableListMultimap.readObject(ImmutableListMultimap.java:375)
15:09:06 java: 	at java.base/java.io.ObjectInputStream.readObject(ObjectInputStream.java:451)
15:09:06 java: 	at java.base/java.io.ObjectInputStream.readObject(ObjectInputStream.java:493)
15:09:06 java: 	at java.base/java.io.ObjectInputStream.readObject0(ObjectInputStream.java:1679)
15:09:06 java: 	at java.base/java.io.ObjectInputStream.readOrdinaryObject(ObjectInputStream.java:2196)
15:09:06 java: 	at java.base/java.io.ObjectInputStream.readSerialData(ObjectInputStream.java:2358)
15:09:06 java: 	at java.base/java.io.ObjectInputStream.defaultReadFields(ObjectInputStream.java:2464)
15:09:06 java: 	at java.base/java.io.ObjectInputStream.readObject0(ObjectInputStream.java:1679)
15:09:06 java: 	at java.base/java.io.ObjectInputStream.readOrdinaryObject(ObjectInputStream.java:2196)
15:09:06 java: 	at java.base/java.io.ObjectInputStream.readSerialData(ObjectInputStream.java:2325)
15:09:06 java: 	at java.base/java.io.ObjectStreamClass.invokeReadObject(ObjectStreamClass.java:1175)
15:09:06 java: 	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
15:09:06 java: 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
15:09:06 java: 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
15:09:06 java: 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
15:09:06 java: 	at java.base/java.util.HashMap.readObject(HashMap.java:1460)
15:09:06 java: 	at java.base/java.io.ObjectInputStream.readObject(ObjectInputStream.java:451)
15:09:06 java: 	at java.base/java.io.ObjectInputStream.readObject(ObjectInputStream.java:493)
15:09:06 java: 	at java.base/java.io.ObjectInputStream.readObject0(ObjectInputStream.java:1679)
15:09:06 java: 	at java.base/java.io.ObjectInputStream.readOrdinaryObject(ObjectInputStream.java:2196)
15:09:06 java: 	at java.base/java.io.ObjectInputStream.readSerialData(ObjectInputStream.java:2358)
15:09:06 java: 	at java.base/java.io.ObjectInputStream.defaultReadFields(ObjectInputStream.java:2464)
15:09:06 java: 	at java.base/java.io.ObjectInputStream.readObject0(ObjectInputStream.java:1679)
15:09:06 java: 	at java.base/java.io.ObjectInputStream.readOrdinaryObject(ObjectInputStream.java:2196)
15:09:06 java: 	at java.base/java.io.ObjectInputStream.readSerialData(ObjectInputStream.java:2325)
15:09:06 java: 	at java.base/java.io.ObjectStreamClass.invokeReadObject(ObjectStreamClass.java:1175)
15:09:06 java: 	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
15:09:06 java: 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
15:09:06 java: 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
15:09:06 java: 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
15:09:06 java: 	at java.base/java.util.ArrayList.readObject(ArrayList.java:929)
15:09:06 java: 	at java.base/java.io.ObjectInputStream.readObject(ObjectInputStream.java:451)
15:09:06 java: 	at java.base/java.io.ObjectInputStream.readObject(ObjectInputStream.java:493)
15:09:06 java: 	at java.base/java.io.ObjectInputStream.readObject0(ObjectInputStream.java:1679)
15:09:06 java: 	at java.base/java.io.ObjectInputStream.readOrdinaryObject(ObjectInputStream.java:2196)
15:09:06 java: 	at java.base/java.io.ObjectInputStream.readSerialData(ObjectInputStream.java:2358)
15:09:06 java: 	at java.base/java.io.ObjectInputStream.defaultReadFields(ObjectInputStream.java:2464)
15:09:06 java: 	at java.base/java.io.ObjectInputStream.readObject0(ObjectInputStream.java:1679)
15:09:06 java: 	at java.base/java.io.ObjectInputStream.readOrdinaryObject(ObjectInputStream.java:2196)
15:09:06 java: 	at java.base/java.io.ObjectInputStream.readSerialData(ObjectInputStream.java:2358)
15:09:06 java: 	at java.base/java.io.ObjectInputStream.defaultReadFields(ObjectInputStream.java:2464)
15:09:06 java: 	at java.base/java.io.ObjectInputStream.readObject0(ObjectInputStream.java:1679)
15:09:06 java: 	at java.base/java.io.ObjectInputStream.readOrdinaryObject(ObjectInputStream.java:2196)
15:09:06 java: 	at java.base/java.io.ObjectInputStream.readSerialData(ObjectInputStream.java:2325)
15:09:06 java: 	at java.base/java.io.ObjectStreamClass.invokeReadObject(ObjectStreamClass.java:1175)
15:09:06 java: 	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
15:09:06 java: 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
15:09:06 java: 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
15:09:06 java: 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
15:09:06 java: 	at java.base/java.util.HashMap.readObject(HashMap.java:1460)
15:09:06 java: 	at java.base/java.io.ObjectInputStream.readObject(ObjectInputStream.java:451)
15:09:06 java: 	at java.base/java.io.ObjectInputStream.readObject(ObjectInputStream.java:493)
15:09:06 java: 	at java.base/java.io.ObjectInputStream.readObject0(ObjectInputStream.java:1679)
15:09:06 java: 	at java.base/java.io.ObjectInputStream.readOrdinaryObject(ObjectInputStream.java:2196)
15:09:06 java: 	at java.base/java.io.ObjectInputStream.readSerialData(ObjectInputStream.java:2358)
15:09:06 java: 	at java.base/java.io.ObjectInputStream.defaultReadFields(ObjectInputStream.java:2464)
15:09:06 java: 	at java.base/java.io.ObjectInputStream.readObject0(ObjectInputStream.java:1679)
15:09:06 java: 	at java.base/java.io.ObjectInputStream.readOrdinaryObject(ObjectInputStream.java:2196)
15:09:06 java: 	at java.base/java.io.ObjectInputStream.readSerialData(ObjectInputStream.java:2325)
15:09:06 java: 	at java.base/java.io.ObjectStreamClass.invokeReadObject(ObjectStreamClass.java:1175)
15:09:06 java: 	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
15:09:06 java: 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
15:09:06 java: 	at java.base/jdk.internal.reflect.GeneratedMethodAccessor15.invoke(Unknown Source)
15:09:06 java: 	at java.base/java.util.ArrayList.readObject(ArrayList.java:929)
15:09:06 java: 	at java.base/java.io.ObjectInputStream.readObject(ObjectInputStream.java:451)
15:09:06 java: 	at java.base/java.io.ObjectInputStream.readObject(ObjectInputStream.java:493)
15:09:06 java: 	at java.base/java.io.ObjectInputStream.readObject0(ObjectInputStream.java:1679)
15:09:06 java: 	at java.base/java.io.ObjectInputStream.readOrdinaryObject(ObjectInputStream.java:2196)
15:09:06 java: 	at java.base/java.io.ObjectInputStream.readSerialData(ObjectInputStream.java:2358)
15:09:06 java: 	at java.base/java.io.ObjectInputStream.defaultReadFields(ObjectInputStream.java:2464)
15:09:06 java: 	at java.base/java.io.ObjectInputStream.readObject0(ObjectInputStream.java:1679)
15:09:06 java: 	at java.base/java.io.ObjectInputStream.readOrdinaryObject(ObjectInputStream.java:2196)
15:09:06 java: 	at java.base/java.io.ObjectInputStream.readSerialData(ObjectInputStream.java:2358)
15:09:06 java: 	at java.base/java.io.ObjectInputStream.defaultReadFields(ObjectInputStream.java:2464)
15:09:06 java: 	at java.base/java.io.ObjectInputStream.readObject0(ObjectInputStream.java:1679)
15:09:06 java: 	at java.base/java.io.ObjectInputStream.readOrdinaryObject(ObjectInputStream.java:2196)
15:09:06 java: 	at java.base/java.io.ObjectInputStream.readSerialData(ObjectInputStream.java:2325)
15:09:06 java: 	at java.base/java.io.ObjectStreamClass.invokeReadObject(ObjectStreamClass.java:1175)
15:09:06 java: 	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
15:09:06 java: 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
15:09:06 java: 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
15:09:06 java: 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
15:09:06 java: 	at java.base/java.util.HashMap.readObject(HashMap.java:1460)
15:09:06 java: 	at java.base/java.io.ObjectInputStream.readObject(ObjectInputStream.java:451)
15:09:06 java: 	at java.base/java.io.ObjectInputStream.readObject(ObjectInputStream.java:493)
15:09:06 java: 	at java.base/java.io.ObjectInputStream.readObject0(ObjectInputStream.java:1679)
15:09:06 java: 	at java.base/java.io.ObjectInputStream.readOrdinaryObject(ObjectInputStream.java:2196)
15:09:06 java: 	at java.base/java.io.ObjectInputStream.readSerialData(ObjectInputStream.java:2358)
15:09:06 java: 	at java.base/java.io.ObjectInputStream.defaultReadFields(ObjectInputStream.java:2464)
15:09:06 java: 	at java.base/java.io.ObjectInputStream.readObject0(ObjectInputStream.java:1679)
15:09:06 java: 	at java.base/java.io.ObjectInputStream.readOrdinaryObject(ObjectInputStream.java:2196)
15:09:06 java: 	at java.base/java.io.ObjectInputStream.readSerialData(ObjectInputStream.java:2325)
15:09:06 java: 	at java.base/java.io.ObjectStreamClass.invokeReadObject(ObjectStreamClass.java:1175)
15:09:06 java: 	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
15:09:06 java: 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
15:09:06 java: 	at java.base/jdk.internal.reflect.GeneratedMethodAccessor16.invoke(Unknown Source)
15:09:06 java: 	at java.base/java.util.HashMap.readObject(HashMap.java:1460)
15:09:06 java: 	at java.base/java.io.ObjectInputStream.readObject(ObjectInputStream.java:451)
15:09:06 java: 	at java.base/java.io.ObjectInputStream.readObject(ObjectInputStream.java:493)
15:09:06 java: 	at java.base/java.io.ObjectInputStream.readObject0(ObjectInputStream.java:1679)
15:09:06 java: 	at java.base/java.io.ObjectInputStream.readOrdinaryObject(ObjectInputStream.java:2196)
15:09:06 java: 	at java.base/java.io.ObjectInputStream.readSerialData(ObjectInputStream.java:2358)
15:09:06 java: 	at java.base/java.io.ObjectInputStream.defaultReadFields(ObjectInputStream.java:2464)
15:09:06 java: 	at java.base/java.io.ObjectInputStream.readObject0(ObjectInputStream.java:1679)
15:09:06 java: 	at java.base/java.io.ObjectInputStream.readOrdinaryObject(ObjectInputStream.java:2196)
15:09:06 java: 	at java.base/java.io.ObjectInputStream.readSerialData(ObjectInputStream.java:2325)
15:09:06 java: 	at java.base/java.io.ObjectStreamClass.invokeReadObject(ObjectStreamClass.java:1175)
15:09:06 java: 	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
15:09:06 java: 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
15:09:06 java: 	at java.base/jdk.internal.reflect.GeneratedMethodAccessor16.invoke(Unknown Source)
15:09:06 java: 	at java.base/java.util.HashMap.readObject(HashMap.java:1458)
15:09:06 java: 	at java.base/java.io.ObjectInputStream.readObject(ObjectInputStream.java:451)
15:09:06 java: 	at java.base/java.io.ObjectInputStream.readObject(ObjectInputStream.java:493)
15:09:06 java: 	at java.base/java.io.ObjectInputStream.readObject0(ObjectInputStream.java:1679)
15:09:06 java: 	at java.base/java.io.ObjectInputStream.readOrdinaryObject(ObjectInputStream.java:2196)
15:09:06 java: 	at java.base/java.io.ObjectInputStream.readSerialData(ObjectInputStream.java:2358)
15:09:06 java: 	at java.base/java.io.ObjectInputStream.defaultReadFields(ObjectInputStream.java:2464)
15:09:06 java: 	at java.base/java.io.ObjectInputStream.readObject0(ObjectInputStream.java:1679)
15:09:06 java: 	at java.base/java.io.ObjectInputStream.readOrdinaryObject(ObjectInputStream.java:2196)
15:09:06 java: 	at java.base/java.io.ObjectInputStream.readSerialData(ObjectInputStream.java:2325)
15:09:06 java: 	at java.base/java.io.ObjectStreamClass.invokeReadObject(ObjectStreamClass.java:1175)
15:09:06 java: 	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
15:09:06 java: 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
15:09:06 java: 	at java.base/jdk.internal.reflect.GeneratedMethodAccessor16.invoke(Unknown Source)
15:09:06 java: 	at java.base/java.util.HashMap.readObject(HashMap.java:1460)
15:09:06 java: 	at java.base/java.io.ObjectInputStream.readObject(ObjectInputStream.java:451)
15:09:06 java: 	at java.base/java.io.ObjectInputStream.readObject(ObjectInputStream.java:493)
15:09:06 java: 	at java.base/java.io.ObjectInputStream.readObject0(ObjectInputStream.java:1679)
15:09:06 java: 	at java.base/java.io.ObjectInputStream.readOrdinaryObject(ObjectInputStream.java:2196)
15:09:06 java: 	at java.base/java.io.ObjectInputStream.readSerialData(ObjectInputStream.java:2358)
15:09:06 java: 	at java.base/java.io.ObjectInputStream.defaultReadFields(ObjectInputStream.java:2464)
15:09:06 java: 	at java.base/java.io.ObjectInputStream.readObject0(ObjectInputStream.java:1679)
15:09:06 java: 	at java.base/java.io.ObjectInputStream.readOrdinaryObject(ObjectInputStream.java:2196)
15:09:06 java: 	at java.base/java.io.ObjectInputStream.readSerialData(ObjectInputStream.java:2325)
15:09:06 java: 	at java.base/java.io.ObjectStreamClass.invokeReadObject(ObjectStreamClass.java:1175)
15:09:06 java: 	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
15:09:06 java: 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
15:09:06 java: 	at java.base/jdk.internal.reflect.GeneratedMethodAccessor15.invoke(Unknown Source)
15:09:06 java: 	at java.base/java.util.ArrayList.readObject(ArrayList.java:929)
15:09:06 java: 	at java.base/java.io.ObjectInputStream.readObject(ObjectInputStream.java:451)
15:09:06 java: 	at java.base/java.io.ObjectInputStream.readObject(ObjectInputStream.java:493)
15:09:06 java: 	at java.base/java.io.ObjectInputStream.readObject0(ObjectInputStream.java:1679)
15:09:06 java: 	at java.base/java.io.ObjectInputStream.readOrdinaryObject(ObjectInputStream.java:2196)
15:09:06 java: 	at java.base/java.io.ObjectInputStream.readSerialData(ObjectInputStream.java:2358)
15:09:06 java: 	at java.base/java.io.ObjectInputStream.defaultReadFields(ObjectInputStream.java:2464)
15:09:06 java: 	at java.base/java.io.ObjectInputStream.readObject0(ObjectInputStream.java:1679)
15:09:06 java: 	at java.base/java.io.ObjectInputStream.readOrdinaryObject(ObjectInputStream.java:2196)
15:09:06 java: 	at java.base/java.io.ObjectInputStream.readSerialData(ObjectInputStream.java:2358)
15:09:06 java: 	at java.base/java.io.ObjectInputStream.defaultReadFields(ObjectInputStream.java:2464)
15:09:06 java: 	at java.base/java.io.ObjectInputStream.readObject0(ObjectInputStream.java:1679)
15:09:06 java: 	at java.base/java.io.ObjectInputStream.readOrdinaryObject(ObjectInputStream.java:2196)
15:09:06 java: 	at java.base/java.io.ObjectInputStream.readSerialData(ObjectInputStream.java:2325)
15:09:06 java: 	at java.base/java.io.ObjectStreamClass.invokeReadObject(ObjectStreamClass.java:1175)
15:09:06 java: 	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
15:09:06 java: 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
15:09:06 java: 	at java.base/jdk.internal.reflect.GeneratedMethodAccessor16.invoke(Unknown Source)
15:09:06 java: 	at java.base/java.util.HashMap.readObject(HashMap.java:1460)
15:09:06 java: 	at java.base/java.io.ObjectInputStream.readObject(ObjectInputStream.java:451)
15:09:06 java: 	at java.base/java.io.ObjectInputStream.readObject(ObjectInputStream.java:493)
15:09:06 java: 	at java.base/java.io.ObjectInputStream.readObject0(ObjectInputStream.java:1679)
15:09:06 java: 	at java.base/java.io.ObjectInputStream.readOrdinaryObject(ObjectInputStream.java:2196)
15:09:06 java: 	at java.base/java.io.ObjectInputStream.readSerialData(ObjectInputStream.java:2358)
15:09:06 java: 	at java.base/java.io.ObjectInputStream.defaultReadFields(ObjectInputStream.java:2464)
15:09:06 java: 	at java.base/java.io.ObjectInputStream.readObject0(ObjectInputStream.java:1679)
15:09:06 java: 	at java.base/java.io.ObjectInputStream.readOrdinaryObject(ObjectInputStream.java:2196)
15:09:06 java: 	at java.base/java.io.ObjectInputStream.readSerialData(ObjectInputStream.java:2325)
15:09:06 java: 	at java.base/java.io.ObjectStreamClass.invokeReadObject(ObjectStreamClass.java:1175)
15:09:06 java: 	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
15:09:06 java: 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
15:09:06 java: 	at java.base/jdk.internal.reflect.GeneratedMethodAccessor16.invoke(Unknown Source)
15:09:06 java: 	at java.base/java.util.HashMap.readObject(HashMap.java:1460)
15:09:06 java: 	at java.base/java.io.ObjectInputStream.readObject(ObjectInputStream.java:451)
15:09:06 java: 	at java.base/java.io.ObjectInputStream.readObject(ObjectInputStream.java:493)
15:09:06 java: 	at java.base/java.io.ObjectInputStream.readObject0(ObjectInputStream.java:1679)
15:09:06 java: 	at java.base/java.io.ObjectInputStream.readOrdinaryObject(ObjectInputStream.java:2196)
15:09:06 java: 	at java.base/java.io.ObjectInputStream.readSerialData(ObjectInputStream.java:2358)
15:09:06 java: 	at java.base/java.io.ObjectInputStream.defaultReadFields(ObjectInputStream.java:2464)
15:09:06 java: 	at java.base/java.io.ObjectInputStream.readObject0(ObjectInputStream.java:1679)
15:09:06 java: 	at java.base/java.io.ObjectInputStream.readOrdinaryObject(ObjectInputStream.java:2196)
15:09:06 java: 	at java.base/java.io.ObjectInputStream.readSerialData(ObjectInputStream.java:2325)
15:09:06 java: 	at java.base/java.io.ObjectStreamClass.invokeReadObject(ObjectStreamClass.java:1175)
15:09:06 java: 	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
15:09:06 java: 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
15:09:06 java: 	at java.base/jdk.internal.reflect.GeneratedMethodAccessor16.invoke(Unknown Source)
15:09:06 java: 	at java.base/java.util.HashMap.readObject(HashMap.java:1461)
15:09:06 java: 	at java.base/java.util.HashMap.hash(HashMap.java:339)
15:09:06 java: 	at games.strategy.engine.data.UnitType.hashCode(UnitType.java:77)
15:09:06 java: java.lang.NullPointerException
```

**Comment by beelee1** [comment]
@STyx2909 ahh bummer. Misunderstood you.

**Comment by STyx2909** [comment]
> @STyx2909 ahh bummer. Misunderstood you.

No, bummer is me. The bug is in front of the computer.
I was expecting a different behaviour from the game.
Everything is OK.

Sorry for bothering you with this.
LOL.

**Comment by DanVanAtta** [comment]
I wonder if there should have been an actual error message instead of the game seemingly tripping up over itself. I didn't quite follow the problem here @STyx2909 , were you moving save games from one OS to another or something like that?

**Comment by STyx2909** [comment]
Do not let me be more ashamed than I am actually. LOL
Ok, so far, I was expecting the game to launch at once after I had picked up a saved file and not to have to click too on "Start local game" to continue playing.
BTW: as I played tutorial 3 times to check this non existing bug, I would like to congratulate the team for it. It is very well done. I enjoyed it every time.
Though it lacks a bit of info on how to load a saved game. ;-)
